numba/tests/test_target_extension.py

```diff
@@ -479,7 +469,9 @@ def cpu_foo():
 
         # DPU chooses the ol_my_func2 as it's most specific, and DPU subtracts
         # for addition, so 10 + x -> 10 - 7 -> 3
-        self.assertPreciseEqual(dpu_foo(), 3)
+        from numba.core.target_extension import target_override
+        with target_override('dpu'):
```

Comment:
Is this needed? `dpu_foo` is `djit` decorated and so shouldn't need an override ctx?

Reply:
Related to this? https://github.com/numba/numba/pull/7061/files#diff-3a1fe479f55566dcfb700cf3ea134261b5bbec513f8433b2393bc3bd7ee46645R731 ?

Reply:
you are right. this isn't needed.

Reply:
Postponing this as discussed OOB.